scalding-core/src/main/scala/com/twitter/scalding/typed/Grouped.scala

```diff
@@ -36,7 +37,7 @@ sealed trait CoGroupable[K, +R] extends HasReducers with HasDescription with jav
   /**
    * This is the list of mapped pipes, just before the (reducing) joinFunction is applied
    */
-  def inputs: List[TypedPipe[(K, Any)]]
+  def inputs(implicit q: Quoted): List[TypedPipe[(K, Any)]]
```

Comment:
can you add a comment why we need the `q` here? This is not a method users should really ever be using (only people writing backends).

Reply:
I had added it because `FilterKeys.input` creates a `TypedPipe`. I reverted these methods and used `Quoted.internal`.